pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -9,8 +9,10 @@ package usm
 
 import (
 	"bytes"
+	"crypto/tls"
 	"encoding/binary"
 	"net"
+	nethttp "net/http"
```

Comment:
Maybe lets change our http use. In HTTP/2 we are using usmhttp2, maybe we should do the same and create usmhttp? instead of nethttp?

Reply:
done